_maps/map_files/RandomRuins/SpaceRuins/abandoned_sec_shuttle.dmm

```diff
@@ -0,0 +1,327 @@
+	dir = 1
+	},
+/turf/simulated/floor/plasteel/airless{
+	icon_state = "dark"
+	},
+/area/ruin/space/sec_shuttle)
+"m" = (
+/obj/effect/decal/cleanable/blood/drip,
+/obj/effect/decal/cleanable/blood,
+/obj/effect/mob_spawn/human/corpse/skeleton/security_officer,
+/turf/simulated/floor/plating/airless,
+/area/ruin/space/sec_shuttle)
+"n" = (
+/obj/effect/decal/cleanable/blood/drip,
+/obj/effect/decal/cleanable/blood/drip,
+/obj/effect/decal/cleanable/blood,
+/turf/simulated/floor/plating/airless,
+/area/ruin/space/sec_shuttle)
+"o" = (
+/obj/structure/window/reinforced{
+	dir = 4
+	},
+/obj/structure/shuttle/engine/heater{
+	dir = 8
+	},
+/turf/space,
+/area/ruin/space/sec_shuttle)
+"t" = (
+/obj/effect/decal/cleanable/blood,
+/obj/effect/decal/cleanable/blood/footprints,
+/turf/simulated/floor/plating/airless,
+/area/ruin/space/sec_shuttle)
+"w" = (
+/obj/structure/chair/comfy/shuttle{
+	dir = 1
+	},
+/turf/simulated/floor/plasteel/airless{
+	icon_state = "dark"
+	},
+/area/ruin/space/sec_shuttle)
+"y" = (
+/obj/effect/decal/cleanable/blood,
+/turf/simulated/floor/plating/airless,
+/area/ruin/space/sec_shuttle)
+"z" = (
+/turf/space,
+/area/template_noop)
+"B" = (
+/obj/machinery/recharge_station/upgraded,
+/turf/simulated/floor/plasteel/airless{
+	icon_state = "dark"
+	},
+/area/ruin/space/sec_shuttle)
+"D" = (
+/obj/machinery/computer,
+/turf/simulated/floor/plasteel/airless{
+	icon_state = "dark"
+	},
+/area/ruin/space/sec_shuttle)
+"E" = (
+/obj/structure/closet/crate/secure/sec_shuttle_ruin{
+	name = "emergency supplies crate"
+	},
+/obj/item/gun/medbeam/damaged,
+/obj/item/gun/projectile/automatic/pistol/enforcer,
+/obj/item/grenade/flashbang,
+/obj/item/storage/firstaid/regular,
+/turf/simulated/floor/plating/airless,
+/area/ruin/space/sec_shuttle)
+"F" = (
+/obj/structure/chair/comfy/shuttle{
+	dir = 1
+	},
+/obj/effect/decal/cleanable/blood/footprints,
+/turf/simulated/floor/plasteel/airless{
+	icon_state = "dark"
+	},
+/area/ruin/space/sec_shuttle)
+"G" = (
+/obj/effect/decal/cleanable/blood/splatter,
+/turf/simulated/floor/plating/airless,
+/area/ruin/space/sec_shuttle)
+"H" = (
+/turf/simulated/floor/plating/airless,
+/area/ruin/space/sec_shuttle)
+"J" = (
+/obj/effect/decal/cleanable/blood/drip,
+/obj/effect/decal/cleanable/blood/footprints,
+/turf/simulated/floor/plating/airless,
+/area/ruin/space/sec_shuttle)
+"N" = (
+/obj/effect/mob_spawn/human/corpse/skeleton/prisoner,
+/turf/space,
+/area/space/nearstation)
+"Q" = (
+/obj/structure/chair/comfy/shuttle{
+	dir = 1
+	},
+/obj/effect/mob_spawn/human/corpse/skeleton/security_officer,
+/turf/simulated/floor/plasteel/airless{
+	icon_state = "dark"
+	},
+/area/ruin/space/sec_shuttle)
+"R" = (
+/obj/structure/chair/comfy/shuttle{
+	dir = 1
+	},
+/turf/simulated/floor/plating/airless,
+/area/ruin/space/sec_shuttle)
+"S" = (
+/obj/machinery/computer/security{
+	dir = 1;
+	network = list("SS13","Telecomms","Research  Outpost","Mining  Outpost","ERT","CentComm","Thunderdome")
```

Comment:
Explorers shouldnt have access to this, even if the ruin is depowered for now, someone could add an APC or something. At most this should be just the ss13 network